src/main.rs

```diff
@@ -1,25 +1,20 @@
-#![warn(missing_docs)]
+#![allow(missing_docs)]
```

Comment:
This was added for a reason

Reply:
oops sorry, that was supposed to be temoprary